docs/source/nlp/question_answering.rst

```diff
@@ -0,0 +1,268 @@
+.. _question_answering:
+
+Question Answering Model
+=====================================================
+
+With the Question Answering, or Reading Comprehension, task, given a question and a passage of
+content (context) that may contain an answer for the question,
+the model will predict the span within the text with a start and end position indicating
+the answer to the question. For datasets like SQuAD 2.0, this model supports cases when the
+answer is not contained in the content.
+
+For every word in the context of a given question, the model will be trained to predict:
+
+- The likelihood this word is the start of the span
+- The likelihood this word is the end of the span
+
+The model chooses the start and end words with maximal probabilities. When the content does not
+contain the answer, we would like the start and end span to be set for the first token.
+
+A pretrained BERT encoder with two span prediction heads is used for the prediction start and
+the end position of the answer. The span predictions are token classifiers consisting of a single
+linear layer.
+
+
```

Comment:
If there are pre-trained models available, could you add a quick start like here https://github.com/NVIDIA/NeMo/blob/r1.0.0rc1/docs/source/nlp/punctuation_and_capitalization.rst#quick-start

Reply:
they are not available yet